acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/privacy/PrivacyGroupAcceptanceTest.java

```diff
@@ -103,6 +103,66 @@ public void nodeCanCreatePrivacyGroup() {
     assertThat(privacyGroups.get(0).getMembers().length).isEqualTo(2);
   }
 
+  @Test
+  public void nodeCanCreatePrivacyGroupWithoutName() {
+    final String privacyGroupId =
+        privacyNet
+            .getNode("Alice")
+            .execute(
+                privateTransactions.createPrivacyGroupWithoutName(
+                    List.of(
+                        privacyNet.getEnclave("Alice").getPublicKeys().get(0),
+                        privacyNet.getEnclave("Bob").getPublicKeys().get(0)),
+                    "my group description"));
+
+    assertThat(privacyGroupId).isNotNull();
+
+    final List<PrivacyGroup> privacyGroups =
+        privacyNet
+            .getNode("Alice")
+            .execute(
+                privateTransactions.findPrivacyGroup(
+                    List.of(
+                        privacyNet.getEnclave("Alice").getPublicKeys().get(0),
+                        privacyNet.getEnclave("Bob").getPublicKeys().get(0))));
+
+    assertThat(privacyGroups.size()).isEqualTo(1);
+    assertThat(privacyGroups.get(0).getPrivacyGroupId()).isEqualTo(privacyGroupId);
+    assertThat(privacyGroups.get(0).getName()).isEqualTo("Default Name");
+    assertThat(privacyGroups.get(0).getDescription()).isEqualTo("my group description");
+    assertThat(privacyGroups.get(0).getMembers().length).isEqualTo(2);
```

Comment:
it may be worth breaking out the participants as a variable, and confirming the content of the privacyGroup.

Reply:
acceptance-tests are being refactored in #1864 I have moved your comments there